web/pandas/static/css/pandas.css

```diff
@@ -103,3 +103,23 @@ blockquote {
     color: #787878;
     font-size: 18px;
  }
+
+.toc {
+    background: #f0f0f0;
+    padding: 10px;
+    display: inline-block;
+}
+a.headerlink {
+    opacity: 0;
+}
+h2:hover a.headerlink {
+    opacity: 1;
+    transition: opacity 0.5s;
+}
+h3:hover a.headerlink {
+    opacity: 1;
+    transition: opacity 0.5s;
+}
```

Comment:
I _think_ we should be able to combine these two, but wasn't able to figure out the syntax.